.github/workflows/deploy.yml

```diff
@@ -1,6 +1,15 @@
 name: Deploy
 
-on: push
+on:
+  workflow_run:
+    workflows:
+      - Test
+    branches:
+      - master
+      - development
```

Comment:
should we make this a staging deployment on `development` branch only? 🤔  production release should be on `master` separately yes 🤔

Reply:
Did you mean create a new workflow to deploy to production separately?
As I have [this step](https://github.com/gutakk/go-google-scraper/pull/20/files#diff-28802fbf11c83a2eee09623fb192785e7ca92a3f40602a517c011b947a1822d3R35-R42) to set the HEROKU_APP by branch

What do you think?

Reply:
yes I think production deployment (on `master`) should have a separate flow and condition from the normal staging deployment 🤔  do you plan to have another workflow separately then? 🤔  if I understand correctly then we may keep this one as-is to run staging deploy on these 2 cases 🙏